.travis.yml

```diff
@@ -1,29 +1,23 @@
 sudo: required
 dist: trusty
-language: generic
+language: c++
 compiler:
   - gcc
   - clang
+notifications:
+  email: false
 env:
-  - CC=gcc CXX=g++
-  - CC=clang CXX=clang++
+  global:
+    - ROS_REPO=ros
+    - CXXFLAGS="-Werror=uninitialized -Werror=return-type -Werror=format"
+    - ADDITIONAL_DEBS=clang
+  matrix:
+    - ROS_DISTRO="indigo" CC=gcc CXX=g++
```

Comment:
duh, thx. Now there are 4 jobs again, not 8 :-) If they build now, I'm happy...

Thx for the help.